sim-cli/src/parsing.rs

```diff
@@ -288,7 +288,7 @@ async fn validate_activities(
                     counter += 1;
                 }
                 unique_name
-            }
+            },
```

Comment:
This commit should be a `fixup` or squashed with the original commit it should belong to (happy for you to do either, since we're early in review).